### PR TITLE
Fixed Issues

### PR DESCRIPTION
**Pull request by RamGoel**
Fixed Issues #32 #23

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,99 +1,82 @@
 <!DOCTYPE html>
```
Comment (pulkit-30) [inline index.html:1]: @RamGoel please revert this file changes
You only need to change or modify dev.html file (as dev.html is primarily used for development and index.html will be updated later)

File: dev.html [hunk 2]
```diff
@@ -1,106 +1,112 @@
...
+      </a>
+      <div class="flex">
+        <a class="headLinks" href="https://twitter.com/nexus_hackathon"><i
+            class=" linkIcon fa-brands fa-twitter"></i></a>
+        <a class="headLinks" href="https://www.instagram.com/nexus_whatthehack/"><i
+            class=" linkIcon fa-brands fa-instagram"></i></a>
+        <a class="headLinks" href="https://www.linkedin.com/company/nexus-what-the-hack/"><i
+            class=" linkIcon fa-brands fa-linkedin-in"></i></a>
+        <a class="headLinks" href=""><i class=" linkIcon fa-brands fa-discord"></i></a>
+        <a class="headLinks" href="mailto:whatthehacknexus@gmail.com"><i class=" linkIcon fa-brands fa-google"></i></a>
```
Comment (pulkit-30) [inline dev.html:44]: Why Google icon? any specific reason @dewank07 @Anv3sh

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

**Comment by pulkit-30** [comment]
Also, where we are using `png` files??

**Comment by pulkit-30** [comment]
@RamGoel there are some errors in the console...

```
Uncaught TypeError: Cannot read properties of null (reading 'classList')
    at HTMLBodyElement.<anonymous> (index.js:120:7)
(anonymous) @ index.js:120
```

Screenshot 2022-08-19 at 7 39 02 PM

File: dev.html (hunk 2 above)
Comment (dewank07) [reply dev.html:44]: I guess for gmail

Comment (pulkit-30) [reply dev.html:44]: ig using mail icon is better 

```
<i class="fa-solid fa-envelope"></i>
```

WDYT @RamGoel

Comment (RamGoel) [reply dev.html:44]: Done, Check the New Pull Request

**Pull request closed without merging**